exercises.ipynb

```diff
@@ -121,51 +301,472 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 169,
    "metadata": {
     "autoscroll": false,
-    "collapsed": false,
     "ein.hycell": false,
     "ein.tags": "worksheet-0",
+    "scrolled": true,
     "slideshow": {
      "slide_type": "-"
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "https://brickset.com/sets/year-2016/page-1\n",
+      "https://brickset.com/sets/year-2016/page-2\n",
+      "https://brickset.com/sets/year-2016/page-3\n",
+      "https://brickset.com/sets/year-2016/page-4\n",
+      "https://brickset.com/sets/year-2016/page-5\n",
+      "https://brickset.com/sets/year-2016/page-6\n",
+      "https://brickset.com/sets/year-2016/page-7\n",
+      "https://brickset.com/sets/year-2016/page-8\n",
+      "https://brickset.com/sets/year-2016/page-9\n",
+      "https://brickset.com/sets/year-2016/page-10\n",
+      "https://brickset.com/sets/year-2016/page-11\n",
+      "https://brickset.com/sets/year-2016/page-12\n",
+      "https://brickset.com/sets/year-2016/page-13\n",
+      "https://brickset.com/sets/year-2016/page-14\n",
+      "https://brickset.com/sets/year-2016/page-15\n",
+      "https://brickset.com/sets/year-2016/page-16\n",
+      "https://brickset.com/sets/year-2016/page-17\n",
+      "https://brickset.com/sets/year-2016/page-18\n",
+      "https://brickset.com/sets/year-2016/page-19\n",
+      "https://brickset.com/sets/year-2016/page-20\n",
+      "https://brickset.com/sets/year-2016/page-21\n",
+      "https://brickset.com/sets/year-2016/page-22\n",
+      "https://brickset.com/sets/year-2016/page-23\n",
+      "https://brickset.com/sets/year-2016/page-24\n",
+      "https://brickset.com/sets/year-2016/page-25\n",
+      "https://brickset.com/sets/year-2016/page-26\n",
+      "https://brickset.com/sets/year-2016/page-27\n",
+      "https://brickset.com/sets/year-2016/page-28\n",
+      "https://brickset.com/sets/year-2016/page-29\n",
+      "https://brickset.com/sets/year-2016/page-30\n",
+      "https://brickset.com/sets/year-2016/page-31\n",
+      "https://brickset.com/sets/year-2016/page-32\n",
+      "https://brickset.com/sets/year-2016/page-33\n",
+      "https://brickset.com/sets/year-2016/page-34\n"
+     ]
+    }
+   ],
    "source": [
     "# Exercise #2\n",
     "\n",
     "# Now write code that gets you all the links from ALL the pages.\n",
     "\n",
-    "# HINT: you will probably want to extract the URL in the \"next\" button on \n",
-    "# the bottom of the search pagination, which looks like \">\".\n",
+    "number_of_pages = list(range(1, 35))\n",
```

Comment:
No!! What if they add more bricksets and there are now 36 pages?? Your code will fail!

Reply:
(Ah, I see now you didn't actually use this, but I leave the comment just to make sure it's clear why it would have been a bad idea ;))